diff -ru a/ext/gettext/config.m4 b/ext/gettext/config.m4
--- a/configure
+++ b/configure
@@ -37797,9 +37797,7 @@ $as_echo "$ext_output" >&6; }
 
 
 if test "$PHP_GETTEXT" != "no"; then
-  for i in $PHP_GETTEXT /usr/local /usr; do
-    test -r $i/include/libintl.h && GETTEXT_DIR=$i && break
-  done
+  GETTEXT_DIR=$PHP_GETTEXT
 
   if test -z "$GETTEXT_DIR"; then
     as_fn_error $? "Cannot locate header file libintl.h" "$LINENO" 5
--- a/ext/gettext/config.m4	2018-11-07 15:35:26.000000000 +0000
+++ b/ext/gettext/config.m4	2018-11-27 00:33:07.000000000 +0000
@@ -6,9 +6,7 @@
 [  --with-gettext[=DIR]      Include GNU gettext support])
 
 if test "$PHP_GETTEXT" != "no"; then
-  for i in $PHP_GETTEXT /usr/local /usr; do
-    test -r $i/include/libintl.h && GETTEXT_DIR=$i && break
-  done
+  GETTEXT_DIR=$PHP_GETTEXT
 
   if test -z "$GETTEXT_DIR"; then
     AC_MSG_ERROR(Cannot locate header file libintl.h)
